components/ui/viewCounter/index.tsx

```
@@ -1,13 +1,18 @@
 import { useEffect, useState } from "react";
 import ChartBarIcon from "@heroicons/react/outline/ChartBarIcon";
 import { env } from "@utils/helpers";
+import type { ViewCounterProps } from "types/common";
 
-const ViewCounter = ({ slug, hideText }) => {
-  const [views, setViews] = useState(0);
-  const [loading, setLoading] = useState(true);
+interface ViewCounterResponse {
+  views: number;
+}
```

ESLint

[error] Do not declare interfaces outside of the /types directory. Centralize all interfaces in /types.

(no-restricted-syntax)